Blocks/URL/test.cc

```diff
@@ -74,7 +74,8 @@ TEST(URLTest, SmokeTest) {
   EXPECT_EQ(8080, u.port);
   EXPECT_EQ("", u.username);
   EXPECT_EQ("", u.password);
-  // NOTE: `ComposeURL` renders default scheme if scheme is empty, not known by port, but host is present (i.e. absolute URLs).
+  // NOTE: `ComposeURL` renders default scheme if scheme is empty, not known by port, but host is present (i.e. absolute
+  // URLs).
```

Comment:
This is one of the many examples where the semantically complete line is separated and will be treated as two independent lines by a line-based diff algorithm or a line-based `NOTE` collector so that the lines will be hard to reconcile because there is no easy programmatic way to determine their relation to each other. The double slash `//` is a line comment, not a block comment, it was meant to be complete on its one line.